### DISCUSSION TITLE
NHibernate prefix usage for NuGet package

### DISCUSSION THREAD

**Discussion by dennis-gr** [nhibernate/nhibernate-core#3250] [General] [answered]
Hi,

I've been working on a MiniProfiler integration for NHibernate which I'd like to publish as a NuGet package named 'NHibernate.MiniProfiler'. I was under the impression that 'NHibernate' is a public prefix that can be used by 3rd parties, but that does not seem to be the case (anymore?). Do I have to use another package name or can you provide me with the permission to publish a package with that name?

Thanks!

**Comment by bahusoid** [comment]
Related discussion https://github.com/nhibernate/nhibernate-core/issues/2969

I was under impression that @hazzik made it public. I still see it's an option in [docs](https://learn.microsoft.com/en-us/nuget/nuget-org/id-prefix-reservation#advanced-prefix-reservation-scenarios):
> During prefix reservation, the owner can choose to make a prefix public. This will still give them the visual indicator showing that the package originates from a reserved prefix, but it will not block future package submissions on the prefix for any owner.

**Comment by dennis-gr** [comment]
Yeah, that's what I got from reading through the issue you linked, but the upload fails (had to check this manually, because dotnet nuget push says the package already exists):

![image](https://user-images.githubusercontent.com/11884104/223428669-f34593f1-307a-45ea-9720-ca6ccecc4046.png)

**Reply by hazzik** [reply, marked as answer]
Should be done now.

**Comment by fredericDelaporte** [comment]
While waiting for solving this, we may work around this limitation by pushing ourselves your package then giving you rights on it on NuGet side.

**Reply by dennis-gr** [reply]
Thanks for the offer, but I'd rather not make things more complicated than necessary.

**Reply by hazzik** [reply]
Should be done now.

**Reply by dennis-gr** [reply]
> Should be done now.

Thank you!
